packages/core/src/api.js

```diff
@@ -121,6 +121,12 @@ export function createPercyServer(percy, port) {
       percy.upload(await WebdriverUtils.automateScreenshot(req.body));
       res.json(200, { success: true });
     })
+  // Recieves events from sdk's.
+    .route('post', '/percy/events', async (req, res) => {
+      const body = percyBuildEventHandler(req, pkg.version);
+      await percy.client.sendBuildEvents(percy.build?.id, body);
```

Comment:
[Optional] or for future, this can be made async. Currently this call is blocking customer script i believe, unless sdks dont wait on the response

Reply:
we can do this in future also. As we are only sending the event from catch block of sdk. (when any screenshot is failing) so it shouldn't have any performance implications.